modules/n1ql/pages/n1ql-language-reference/build-index.adoc

```diff
@@ -1,30 +1,33 @@
 = BUILD INDEX
 :page-topic-type: concept
+:imagesdir: ../../assets/images
 
 [abstract]
 The BUILD INDEX statement enables you to build one or more GSI indexes that are marked for deferred building all at once.
 
 By default, CREATE INDEX statement starts building the created index.
-However for more efficient building of multiple indexes, CREATE INDEX command can mark indexes for deferred building using the "defer_build:true" option.
+However for more efficient building of multiple indexes, CREATE INDEX command can mark indexes for deferred building using the `defer_build:true` option.
 BUILD INDEX is capable of building multiple indexes all at once and can utilize a single scan to feed many index build operations.
```

Comment:
It would be better to change "single scan" to "single scan of bucket" or "single scan of documents in the bucket".

Reply:
Done